lib/pages/scan/companies_list.dart

```diff
@@ -13,35 +13,57 @@ class CompaniesList extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final int listSize = state.list.length;
+
     _scrollToTop();
-    return Column(mainAxisAlignment: MainAxisAlignment.end, children: <Widget>[
-      Container(
-        height: 200,
-        child: Align(
-          alignment: Alignment.bottomCenter,
-          child: ListView.builder(
-            controller: listScrollController,
-            reverse: true,
-            itemCount: state.list.length + (state.isLoading ? 1 : 0),
-            itemBuilder: (BuildContext context, int index) {
-              if (index == state.list.length) {
-                return LoadingListItem();
-              }
-              return GestureDetector(
-                child: ResultListItem(state.list[index]),
-                onTap: () {
-                  final result = state.list[index];
-                  _analytics.opensCard(result);
-                  Navigator.pushNamed(context, '/detail', arguments: result);
-                },
-              );
-            },
+
+    double maxHeight;
+    if (listSize == 1) {
```

Comment:
Czemu tak? Robiliśmy tydzień temu to w jednej linijce. Co prawda był tam błąd, ale tu też jest no i nadal można było to ograć. Jakbyś wysłał od razu zmiany w PRze to byśmy szybciej ograli. Błąd jest taki, że dla pustej listy będzie wysokość ustawiana na 190.


Można to ograć tak `double maxHeight = min(listSize * 47.5, 190.0)`. W tym przypadku będzie liczymy wysokośc na podstawie liczby elementów. Jeśli wyjdzie większa niż 190 to wysokość będzie 190.